### DISCUSSION TITLE
为啥我播放音乐之后点击链接音乐就会重新开始播放

### DISCUSSION THREAD

**Discussion by chumho** [volantis-x/hexo-theme-volantis#676] [Q&A]
版本是volantis-5.0.0-rc.1，以前volantis 3倒是没啥问题

**Comment by BeaCox** [comment]
不知道你的音樂播放器是在頁脚，側邊欄還是導航欄呢？
pjax開啓了沒有？
信息太少很難判斷
之前使用該版本是能正常使用音樂播放器的，請查閲文檔、檢查配置項
